### PR TITLE
Add find corporate record method

### PR DESCRIPTION
**Pull request by bshyong**
- Add method for finding records based on first/last name
- bump gem version

Gem publication to be done after merge

### CODE REVIEW

File: lib/lighthouse_bgs/services/vet_record.rb [hunk 1]
```diff
@@ -36,5 +36,20 @@ def update_birls_record(ssn:, poa_code:)
...
+    def find_corporate_record(first_name:, last_name:)
+      response = request(
+        :find_corporate_record,
+        {
+          "ptcpntSearchPSNInput": {
+            "firstName": first_name,
+            "lastName": last_name
+          }
+        },
+        ssn
```
Comment (jjulian) [inline lib/lighthouse_bgs/services/vet_record.rb]: I don't think we need to pass this

Comment (charleystran) [reply lib/lighthouse_bgs/services/vet_record.rb]: if `mock_responses` is false it wont matter, but if its true, then this arg will be used to identify the mock data file to pull

Comment (jjulian) [reply lib/lighthouse_bgs/services/vet_record.rb]: Also, there is no `ssn` arg in this method

**Comment by bshyong** [comment]
Making a note to look at mocking for this new method—the 3rd argument is needed to determine which Mock Response file to pull up when mocking is turned on

**Pull request merged**